package.json

```diff
@@ -66,7 +66,10 @@
     "yargs": "^17.0.1"
   },
   "devDependencies": {
+    "@types/electron-packager": "^15.0.1",
```

Comment:
Electron-packager ships its own types since https://github.com/electron/electron-packager/releases/tag/v15.0.0 . When it's the case, we want to prefer shipped-by-packages types than third-party DT types. Or are you having trouble with the types shipped by the package itself?

Reply:
See above